roles/fio_distributed/templates/servers.yaml

```diff
@@ -59,15 +59,23 @@ spec:
           path: {{ hostpath }}
           type: DirectoryOrCreate
 {% endif %}
-{% if metadata is defined and metadata.collection|default(false) is sameas true and metadata.targeted|default(true) is sameas true %}
+{% if metadata is defined and (metadata.collection|default(false)) and (metadata.targeted|default(true)) %}
       initContainers:
       - name: backpack-{{ trunc_uuid }}
         image: {{ metadata_image | default('quay.io/cloud-bulldozer/backpack:latest') }}
         command: ["/bin/sh", "-c"]
-{% if metadata is defined and metadata.force|default(false) is sameas true %}
-        args: ["python3 stockpile-wrapper.py -s {{ elasticsearch.server }} -p {{ elasticsearch.port }} -u {{ uuid }} -n $my_node_name -N $my_pod_name --redisip {{ bo.resources[0].status.podIP }} --redisport 6379 --force"]
-{% else %}
-        args: ["python3 stockpile-wrapper.py -s {{ elasticsearch.server }} -p {{ elasticsearch.port }} -u {{ uuid }} -n $my_node_name -N $my_pod_name --redisip {{ bo.resources[0].status.podIP }} --redisport 6379"]
+        args:
```

Comment:
:) should all the stockpile runs (across benchmarks) be updated to be "pretty"

Reply:
fixed by https://github.com/cloud-bulldozer/ripsaw/pull/383